code/game/machinery/computer/card.dm

```diff
@@ -756,6 +756,14 @@ GLOBAL_VAR_INIT(time_last_changed_position, 0)
 	req_access = list(ACCESS_CMO)
 	circuit = /obj/item/circuitboard/card/minor/cmo
 
+/obj/machinery/computer/card/minor/qm
+	name = "supply management console"
+	target_dept = TARGET_DEPT_SUP
+	icon_screen = "idsup"
+	light_color = LIGHT_COLOR_RED
```

Comment:
Light color red? Is this intended?

Reply:
As far as I can tell the light color doesn't actually do anything or change anything with this console, but I'll check and get back to you.

Reply:
So after some testing, the light_color var doesn't seem to really matter at all, and is leftover from an old PR. See the following images. One is with me varediting the light range to the value of 10, up from its current value. 

![image](https://github.com/ParadiseSS13/Paradise/assets/85680653/718c763e-6d4a-4047-bfac-1a9fcc84a33b)

As you can see, it doesn't produce any red light until the range is VV edited.

![image](https://github.com/ParadiseSS13/Paradise/assets/85680653/8b42b8ce-16fd-448d-907c-d4a045acfa35)

In which case it has a very dull red colour to it.

Reply:
I would still change it to yellow just in case

Reply:
![image](https://github.com/ParadiseSS13/Paradise/assets/85680653/201541c7-7ab4-40e8-884f-05131e20eaba)

Changed to a more fitting colour in the next commit. (This is the console with the light range and power upped way high)